NEW_PROJECT_APPLICATION.md

```diff
@@ -13,12 +13,20 @@ A rough description of the project in less than 100 words.
 Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
 Please keep your answer to less than 250 words.
 
+## Impact and consumers of the project
+- What is the ecosystem impact of the project? 
+- Who are the biggest consumers of the project? 
+_if they are not JavaScript space, then will it ever expand to JavaScript space_.
```

Comment:
Need a bullet here? Also I'm wondering if we'd consider a project that wasn't using JavaScript.

Reply:
>Need a bullet here? 

It is a continuation and not a separate question.

Reply:
> Also I'm wondering if we'd consider a project that wasn't using JavaScript.

This should probably be addressed. I'm not sure the answer here either.

Reply:
> It is a continuation and not a separate question.

Perhaps this should be added into the question to help clarify that instead of having it on a separate line? So something like `Who are the biggest consumers of the project and if they are not JavaScript space, then will it ever expand to JavaScript space?`

> Also I'm wondering if we'd consider a project that wasn't using JavaScript.

This would depend on what you mean by "using" JavaScript. IMO, anything with some sort of relationship with the JavaScript ecosystem would be a potential candidate. For example, libuv is already an OpenJSF project but is not solely a JavaScript-focused project and does not have JS in it's source (or at least not that I am aware of in the limited knowledge I have of that project's inner workings).